experimental/language-generation/csharp_dotnetcore/05.multi-turn-prompt/Dialogs/UserProfileDialog.cs

```diff
@@ -9,22 +9,21 @@
 using Microsoft.Bot.Builder.Dialogs;
 using Microsoft.Bot.Builder.Dialogs.Choices;
 using Microsoft.Bot.Builder.LanguageGeneration;
-using Microsoft.Bot.Builder.Dialogs.Adaptive.Generators;
 
 namespace Microsoft.BotBuilderSamples
 {
     public class UserProfileDialog : ComponentDialog
     {
         private readonly IStatePropertyAccessor<UserProfile> _userProfileAccessor;
-        private static TemplateEngine _lgEngine;
+        private static Templates _lgTemplates;
```

Comment:
>_lgTemplates [](start = 33, length = 12)

I would call just _templates. lgTemplates looks like we are doing hungarian notation, and templates is the class name.

Reply:
Fixed.